quickjs.h

```diff
@@ -30,6 +30,7 @@
 #include <stdio.h>
 #include <stdint.h>
 #include <string.h>
+#include <math.h>
```

Comment:
Simple fix! I'm surprised this problem did not surface before. Note however that `NAN` is also defined in `<float.h>` as specified in **5.2.4.2.2 Characteristics of floating types <float.h>** and the standard says:
> Use of the macros `INFINITY`, `DEC_INFINITY`, `NAN`, and `DEC_NAN` in `<math.h>` is an obsolescent
feature. Instead, use the same macros in `<float.h>`.

So I sugest including `<float.h>` instead of `<math.h>`